core/injector/application_injector.hpp

```diff
@@ -464,6 +467,10 @@ namespace kagome::injector {
         // bind io_context: 1 per injector
         di::bind<::boost::asio::io_context>.in(
             di::extension::shared)[boost::di::override],
+        di::bind<api::RpcContext>.in(
+            di::extension::shared)[boost::di::override],
```

Comment:
I think override is redundant here as this is the only place where bind is applied to RpcContext

Reply:
Sure. Will be removed.